rq/utils.py

```diff
@@ -230,7 +230,7 @@ def str_to_date(date_str):
     if not date_str:
         return
     else:
-        return utcparse(date_str.decode())
+        return utcparse(as_text(date_str))
```

Comment:
`as_text` is a Python 2/3 compatibility function that will be removed in the future. Could you build string handling directly in this function instead of using `as_text`?